### PR TITLE
1.20

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

This pr adds 1.20 support 
This went suprisingly easy
I have tested this with networks no errors.

@Boomer-1 is gonna test for me even more

## Proposed changes

Added 1.20 to the supported versions
Changed the pom to 1.20

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [ x I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.16.* - 1.19.*).
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Comment by ybw0014** [comment]
unit test failed :eyes:

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import org.bukkit.Bukkit;
 import org.bukkit.Server;
 import org.bukkit.World;
+import org.bukkit.block.Biome;
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java:22]: An unused import?

File: src/test/resources/biomes/1.20.x.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+[
+	"minecraft:ocean",
+	"minecraft:plains",
+	"minecraft:desert",
+	"minecraft:windswept_hills",
+	"minecraft:forest",
+	"minecraft:taiga",
+	"minecraft:swamp",
+	"minecraft:mangrove_swamp",
+  "minecraft:river",
```
Comment (TheBusyBiscuit) [inline src/test/resources/biomes/1.20.x.json:10]: The formatting seems off here.

**Review by TheBusyBiscuit — COMMENTED** [review]
Looks alright to me, just two general comments.
Is this all that's needed to be updated?

Also thank you very much for this PR :heart:

**Review by TheBusyBiscuit — APPROVED** [review]
LGTM if nothing else is needed and no inherent issues are found while testing

**Review by Sfiguz7 — DISMISSED** [review]
LGTM

**Comment by J3fftw1** [comment]
> dough PR was merged so this should be ready to go

No touchy touchy till I Updaty Updaty dough

**Comment by ybw0014** [comment]
I guess we need to change how Slimefun deal with sf item use in the new smithing table GUI in 1.20. maybe in a separate PR.

**Comment by J3fftw1** [comment]
> I guess we need to change how Slimefun deal with sf item use in the new smithing table GUI in 1.20. maybe in a separate PR.

This PR is just gonna focus on supporting 1.20.
I do wanna raise an issue this weekend by toggling of the legacy enum in the spigot.yml. I have feeling that is gonna cause a lot of errors.

**Review by WalshyDev — APPROVED** [review]

**Review by Sefiraat — APPROVED** [review]
👍

**Review by svr333 — APPROVED** [review]

**Review by JustAHuman-xD — APPROVED** [review]
LGTM

**Pull request merged**